charts/everest/charts/common/templates/_db_resources_cleanup.yaml.tpl

```diff
@@ -56,6 +61,7 @@ metadata:
   annotations:
     "helm.sh/hook": pre-delete
     "helm.sh/hook-delete-policy": before-hook-creation,hook-succeeded
+    "helm.sh/hook-weight": "-1"
```

Comment:
Helm sorts the hooks based on the ascending order of the weight, so in this case, `-1` runs before all